lib/DDG/Spice/Coupons.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Coupons;
+
+# ABSTRACT: Returns coupon codes for various online-shops from sparheld.de
+
+use DDG::Spice;
+
+# Attribution
+name "Coupons";
+source "www.sparheld.de";
+description "Get rebate codes from www.sparheld.de";
+category "special";
+topics "everyday";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Coupons.pm";
+attribution github => ['https://github.com/meingutscheincode/zeroclickinfo-spice','Sparheld International GmbH'];
+primary_example_queries "gutschein zalando";
+secondary_example_queries "gutschein h&m";
+
+
+# Caching - yes
+spice is_cached => 1;
```

Comment:
We likely don't want this caching enabled because a search for "gutschein h&m" isn't guarenteed to need the same API call, as it won't always come from the same locale. We want to execut the `handle` each time we get this query, to ensure we get the searcher's Locale and make the correct API call.